### PR TITLE
[Improve] Optimize log display during startup command

### PR DESCRIPTION
**Pull request by JohnZp**
### Purpose of this pull request

Some config loading log don't take effect about log4j2
When we startup a command like seatunnel.sh -l, the log as follows.

image

Generally, if you execute commands through some programs, such non-standard logs will be captured in the error log stream, which is very bad
### Does this PR introduce _any_ user-facing 
image

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Thanks @JohnZp for created this PR! 
~~Could you provide a demo of after optimized?~~
ignore it.

File: seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.cmd [hunk 1]
```diff
@@ -62,7 +62,7 @@ REM port in your IDE. After that, you can happily debug your code.
 REM set "JAVA_OPTS=%JAVA_OPTS% -Xdebug -Xrunjdwp:server=y,transport=dt_socket,address=5001,suspend=y"
 
 if exist "%CONF_DIR%\log4j2.properties" (
-    set "JAVA_OPTS=%JAVA_OPTS% -Dlog4j2.configurationFile=%CONF_DIR%\log4j2.properties"
+    set "JAVA_OPTS=%JAVA_OPTS% -Dhazelcast.logging.type=log4j2 -Dlog4j2.configurationFile=%CONF_DIR%\log4j2.properties"
```
Comment (Hisoka-X) [inline seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.cmd:65]: cc @hailin0

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by wuchunfu — APPROVED** [review]

File: seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.cmd (hunk 1 above)
Comment (hailin0) [reply seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.cmd:65]: Sorry I'm late, this is a good pr

**Comment by hailin0** [comment]
LGTM

**Pull request merged**